notifications_api_common/templates/notifications_api_common/master.html

```diff
@@ -0,0 +1,88 @@
+  </head>
+  <body>
+
+    {% if not settings.SITE_TITLE or not settings.PROJECT_NAME %}
+      <div class="container">
+        <div class="alert alert-danger">
+          Please add the variables <em>SITE_TITLE</em> and <em>PROJECT_NAME</em> to your <em>settings.py</em> and make
+          sure they are available via <em>context processors</em>.
+        </div>
+      </div>
+    {% endif %}
+
+    {% block content %}
+      Override <code>{% verbatim %}{% block content %}{% endverbatim %}</code>
+      to get some actual content.
+    {% endblock %}
+
+    <div class="container">
+      <footer class="pt-4 my-md-5 pt-md-5 border-top">
+        <div class="row">
+          <div class="col-12 col-md">
+            <img class="mb-2" src="{% static 'notifications_api_common/img/vng.svg' %}" alt="" width="96" height="50">
+            <small class="d-block mb-3 text-muted">&copy; {% now "Y" %}</small>
+          </div>
+          <div class="col-6 col-md">
+            <h5>Resources</h5>
+            <ul class="list-unstyled text-small">
+              <li><a class="text-muted" href="https://github.com/{{settings.SELF_REPO}}">Code op Github</a></li>
+              <li><a class="text-muted" href="https://github.com/{{settings.SELF_REPO}}/blob/master/README.rst">README</a></li>
+              <li><a class="text-muted" href="https://zaakgerichtwerken.vng.cloud/">Documentatie</a></li>
+              <li><a class="text-muted" href="https://zaken-auth.vng.cloud">Autorisaties</a></li>
+              <li><a class="text-muted" href="https://zgw-demo.vng.cloud">Demo app</a></li>
+            </ul>
+          </div>
+          <div class="col-6 col-md">
+            {% if not request.get_host|is_local %}
+                <h5>Terms of use</h5>
+                <ul class="list-unstyled text-small">
+                  <li><a class="text-muted" href="https://zaakgerichtwerken.vng.cloud/beheer/gebruiksvoorwaarden">Terms</a></li>
+                </ul>
+            {% endif %}
+          </div>
+          <div class="col-6 col-md">
+            <h5>Over Common Ground</h5>
+            <ul class="list-unstyled text-small">
+              <li><a class="text-muted" href="https://commonground.nl/">Common Ground</a></li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-6 text-muted">
+            <code>{{ settings.API_VERSION }}</code>
+          </div>
+          <div class="col-6 text-muted text-right">
+            <code>{{ settings.GIT_SHA|default:"" }}</code>
+          </div>
+        </div>
+      </footer>
+    </div>
+
+    <script src="{% static 'notifications_api_common/libs/jquery/jquery-3.3.1.slim.min.js' %}"></script>
+    <script src="{% static 'notifications_api_common/libs/popper/popper.min.js' %}"></script>
+    <script src="{% static 'notifications_api_common/libs/bootstrap/js/bootstrap.min.js' %}"></script>
```

Comment:
This still introduceds/requires custom settings and probably contains a bunch of links that are not really necessary.

Can't we have the `KanalenView` use a template, which you can and should override per project? So then we can drop all the static assets from notifications-api-common, as they add a lot of bloat (and duplicate files with vng-api-common).